server/src/components.yaml

```diff
@@ -25,50 +23,47 @@ components:
         discipline:
```

Comment:
So every user will need a discipline (before registering?)

Reply:
The code which we copied over from last year to generate the frosh groups requires that everyone has a discipline at the time they register, so currently yes. Do you think it is worth changing this?

Reply:
No, it's actually fine because it will make the actual registration have less steps
Which ones are currently needed on making an account?

By the looks of it is it
Email
Name
Password
Discipline
Pronouns

Reply:
Just email, name, password.

Discipline and Pronouns are needed when registering

Reply:
Ohh i see I thought it was for just creating an account
Okay thats fine then